hist/hist/inc/LinkDef.h

```diff
@@ -147,7 +147,9 @@
 #pragma link C++ class THnSparse+;
 #pragma link C++ class THnSparseT<TArrayD>+;
 #pragma link C++ class THnSparseT<TArrayF>+;
+#pragma link C++ class THnSparseT<TArrayL>+;
```

Comment:
We might still be missing a deprecation warning.

Reply:
Where/how should I add that? We have so many ways to deprecate old classes, and this is not really deprecated to be fair.